syft/source/directory_resolver.go

```diff
@@ -202,6 +209,7 @@ func (r directoryResolver) addPathToIndex(p string, info os.FileInfo) (string, e
 		r.refsByMIMEType[metadata.MIMEType] = append(r.refsByMIMEType[metadata.MIMEType], *ref)
 	}
 
+	log.Infof("indexed filesystem path=%q", p)
```

Comment:
this log line is pretty verbose for the execution path it's in --was it meant to be used for troubleshooting then removed?